rules_building_block/persistence_startup_folder_lnk.toml

```diff
@@ -33,8 +33,10 @@ file where host.os.type == "windows" and event.type != "deletion" and file.exten
     "C:\\ProgramData\\Microsoft\\Windows\\Start Menu\\Programs\\StartUp\\*"
   ) and
   not (
-    (process.name : "ONENOTE.EXE" and process.code_signature.status: "trusted" and file.name : "Send to OneNote.lnk") or
-    (process.name: "OktaVerifySetup.exe" and process.code_signature.status: "trusted" and file.name : "Okta Verify.lnk")
+    (process.name : "ONENOTE.EXE" and process.code_signature.status: "trusted" and file.name : "*OneNote.lnk") or
```

Comment:
Don't we have `file.path`s that are predictable here to not rely on starting wildcards?

Reply:
don't think EG has code signature info for file event ?

Reply:
> Don't we have file.paths that are predictable here to not rely on starting wildcards?

The file.name here is dependent on the system language, so we have values like: `Invia a OneNote.lnk`, `Send til OneNote.lnk`, `Envoyer à OneNote.lnk`, `Send to OneNote.lnk`, etc

Reply:
> don't think EG has code signature info for file event ?

Hmmm, that is right, do you think we should remove the process signature check or endgame compat?

Reply:
++ EG compat is better as OneNote process path may require extra wildcards for users homeprofile